packages/swarm-components/src/TextInput.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Odd -- these TextInput files were merged to master an hour ago via this https://github.com/meetup/swarm-ui/pull/87/ PR. However, they seem to have been lost from master since then. I'm just going to remerge here as part of this.